examples/readme/modules/builtins.md

```diff
@@ -8,6 +8,7 @@ They are already included when we use this package.
 - `files.cmds`, install packages from [nix repository](https://search.nixos.org/)
 - `files.docs`, convert our modules file into markdown using [nmd](https://gitlab.com/rycee/nmd)
 - `files.git`, configure git with file creation
+- `files.on-call`, connfigure file to created only when devshell-files command is called, not on shell start
```

Comment:
@adrian-gierakowski I'm not good at naming things, suggestions?

Reply:
1. `create-on-enter`
2. `create-on-startup`
3. `create-on-shell-start`

Maybe should be `camelCase` though? The case usage seems a bit inconsistent in nixos module options, but I'd usually use `kebab-case` only when the value is expected to be a package, since the name would often correspond to executable name.